### PR TITLE
Use permission client lib

### PR DESCRIPTION
**Pull request by piuccio**
@jamespamplin

### CODE REVIEW

File: facia-tool/app/conf/PermissionsCache.scala [hunk 1]
```diff
@@ -8,8 +8,7 @@ trait PermissionsCache extends GlobalSettings {
   override def onStart(app: Application) {
     super.onStart(app)
     Logger.info("starting permissions cache")
-    val job = new PermissionsJob()
-    job.start()
+    val config = Permissions.config
```
Comment (jamespamplin) [inline facia-tool/app/conf/PermissionsCache.scala]: Does this get you around the initialisation issue we saw?

**Comment by jamespamplin** [comment]
:+1: great stuff!

File: facia-tool/app/conf/PermissionsCache.scala (hunk 1 above)
Comment (piuccio) [inline facia-tool/app/conf/PermissionsCache.scala]: yup it does. I'll find a way to clean this later

Comment (jamespamplin) [inline facia-tool/app/conf/PermissionsCache.scala]: :+1: no worries, I need to figure out how to fix the issue in the lib too

**Pull request merged**